modules/Microsoft.EventGrid/subscriptions/deploy.bicep

```diff
@@ -0,0 +1,82 @@
+@description('Required. The name of the Event Grid Topic.')
+param name string
+
+@description('Optional. Location for all Resources.')
+param location string = resourceGroup().location
+
+@description('Optional. Enable telemetry via a Globally Unique Identifier (GUID).')
+param enableDefaultTelemetry bool = true
+
+@description('Required. Name of the Event Grid Topic.')
+param eventGridTopicName string
+
+@description('Optional. Dead Letter Destination. (See https://learn.microsoft.com/en-us/azure/templates/microsoft.eventgrid/eventsubscriptions?pivots=deployment-language-bicep#deadletterdestination-objects for more information).')
+param deadLetterDestination object = {}
+
+@description('Optional. Dead Letter with Resource Identity Configuration. (See https://learn.microsoft.com/en-us/azure/templates/microsoft.eventgrid/eventsubscriptions?pivots=deployment-language-bicep#deadletterwithresourceidentity-objects for more information).')
+param deadLetterWithResourceIdentity object = {}
+
+@description('Optional. Delivery with Resource Identity Configuration. (See https://learn.microsoft.com/en-us/azure/templates/microsoft.eventgrid/eventsubscriptions?pivots=deployment-language-bicep#deliverywithresourceidentity-objects for more information).')
+param deliveryWithResourceIdentity object = {}
+
+@description('Required. The destination for the event subscription. (See https://learn.microsoft.com/en-us/azure/templates/microsoft.eventgrid/eventsubscriptions?pivots=deployment-language-bicep#eventsubscriptiondestination-objects for more information).')
+param destination object
+
+@description('Optional. The event delivery schema for the event subscription.')
+param eventDeliverySchema string = 'EventGridSchema'
+
+@description('Optional. The expiration time for the event subscription.')
```

Comment:
Can you please add something like (e.g. XXXXXX) what should the expected value look like?

Reply:
Added to deploy.test.bicep - and added format to description, not adding default value as its a static ISO6801 value